src/content/docs/discovery/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview
+description: Learn about Adobe Commerce on Edge Delivery Services project requirements, data validation, dropins, and Luma Bridge.
+---
+
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+
+Before starting any Adobe Commerce on Edge Delivery Services project, you must conduct a discovery phase to scope the project and ensure that there are no major roadblocks or risks.
+
+## Big picture
+
+In the [create your storefront](/get-started/) tutorial, you learned how to quickly create a new project based on the [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce). The boilerplate provides a starter storefront that uses a pre-configured Adobe Commerce environment.
+
+The next step is to plan, build, optimize, and launch a production-ready storefront connected to your own Adobe Commerce instance and storefront services. Here's an overview of the process:
+
+<Diagram caption="Storefront project planning and delivery process.">
+  ![Storefront project planning and delivery process.](@images/implementation/implementation-overview-v1.svg)
+</Diagram>
+
+<Callouts>
+
+1. **Discovery**: Understand the requirements and goals of the project.
+1. **Setup**: Configure the project environment and tools.
+1. **Marketing and SEO**: Optimize the storefront for search engines and marketing campaigns.
+1. **Launch**: Deploy the storefront to a production environment.
+
+</Callouts>
+
+The discovery phase is important because Adobe Commerce is a highly customizable platform with a large thid-party extension ecosystem. Migrating an Adobe Commerce storefront to Edge Delivery Services is similar to migrating to a headless storefront (like [PWA Studio](https://developer.adobe.com/commerce/pwa-studio)).
+
+:::note[Important]
+The key requirement for a headless storefront implementation is that all required data is provided by APIs. Adobe Commerce provides all of the API coverage that you need for a successful implementation.
+:::
+
+## Existing storefronts
+
+If you have an existing Adobe Commerce storefront (Luma or PWA Studio), you can migrate it to Edge Delivery Services. The migration process involves identifying:
```

Comment:
* ** Project Scope**: Identify use cases to be implemented in the new storefront and those that existing storefront will continue to be responsible for. E.g. Catalog, product detail, cart, checkout, customer  account 
* **Third-party extensions**: Identify list of extensions that you will need on your new storefront and if they rely on integrating 3rd party services 
* **Storefront Bridge options**:  plan to connect existing storefront with a new one. if you are using Adobe Commerce native storefront with Luma,  you can use the [Luma Bridge](/discovery/luma-bridge/)

Reply:
### Full implementation 
For the full implementation the list of considerations is very similar: 
* **Project scope**: Identify use cases new storefront to support. Are there Commerce GraphQL APIs enabled for them.
* **Third-party extensions**: Identify list of extensions that you will need on your new storefront and if they rely on integrating 3rd party services